api/src/services/googleapi/googleapi.js

```diff
@@ -0,0 +1,214 @@
+import { google } from 'googleapis'
+import fs from 'fs'
+
+const createGoogleDriveClient = async () => {
+  const auth = new google.auth.GoogleAuth({
+    credentials: {
+      client_email: process.env.CLIENT_EMAIL,
+      private_key: process.env.PRIVATE_KEY.replace(/\\n/gm, '\n'),
+    },
+    scopes: ['https://www.googleapis.com/auth/drive'],
+  })
+
+  const authClient = await auth.getClient()
+
+  const driveClient = google.drive({
+    version: 'v3',
+    auth: authClient,
+  })
+
+  return driveClient
+}
+
+const createGoogleSheetClient = async () => {
+  const auth = new google.auth.GoogleAuth({
+    credentials: {
+      client_email: process.env.CLIENT_EMAIL,
+      private_key: process.env.PRIVATE_KEY.replace(/\\n/gm, '\n'),
+    },
+    scopes: ['https://www.googleapis.com/auth/spreadsheets'],
+  })
+
+  const authClient = await auth.getClient()
+
+  const sheetClient = google.sheets({
```

Comment:
If we really want, we can make this shorter.

```suggestion
return google.sheets({
  version: 'v4',
  auth: await auth.getClient(),
});
```